aries-site/src/pages/design-tokens/layout-and-spacing.mdx

```diff
@@ -0,0 +1,93 @@
+import { medium } from 'hpe-design-tokens';
+import { Box, Text } from 'grommet';
+import { Checkmark } from 'grommet-icons';
+import { ContentPane, BestPracticeGroup, Example } from '../../layouts';
+import { DesignTokensTable } from '../../components';
+import { SpacingBestPractices } from '../../examples';
+
+The proximity and grouping of elements establish a clear content hierarchy that facilitates a user in accomplishing their goals, without additional distractions.
+
+Layouts should be:
+
+- **Responsive**: They should adapt to different screen sizes and device types.
+- **Consistent**: They should maintain a predictable structure across the application.
+- **Semantic**: They should be designed with HTML semantics in mind to ensure accessibility.
+
+## Foundations
+
+### Sizing system
+
+The sizing scale consists of a primary scale based on the doubling/halving of a 24px base unit and a supplementary scale doubling/halving a 16px base unit.
+
+This results in the following: 1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 192, 256, 384, 512, 768, 1024, 1536.
+
+Spacing, container, radius, and borderWidth tokens are created using a subset of this scale.
+
+## Spacing
+
+Spacing uses proximity principles to create a rhythm and hierarchy within a page. Spacing design tokens define the padding within, margin around, or gap between content.
+
+Rather than relying heavily on borders to separate content, spacing creates a visual hierarchy that guides the user’s eye through the page and helps them understand the relationships between different elements.
+
+There are 12 spacing tokens to provide flexibility to cater to various design needs ranging from dense data-rich experiences to more spacious layouts.
+
+<ContentPane>
+  <DesignTokensTable active="semantic.spacing" toolbar />
+</ContentPane>
+
+### Spacing best practices
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Use spacing design tokens to style compositional or layout components like cards, layers, and grid spacing.',
+    }}
+    width="100%"
+  >
+    <SpacingBestPractices />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Use spacing design tokens to style element-level components. Instead use component-specific tokens or element tokens which will ensure correct sizing of elements across screen sizes.`,
+    }}
+  >
+    <SpacingBestPractices bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+## Containers
+
+Containers define the minimum/maximum width or height of a page section.
+
+Container tokens are particularly useful when defining grid layouts. In other cases, consider the approach of [content-driven layouts](/templates/content-layouts#content-driven-layouts) to avoid artificially constraining content.
+
+There are TBD container tokens to provide flexibility to cater to various design needs.
```

Comment:
so coming soon, or is this still up in the air?

Reply:
Good catch! They've been decided, I'll add in the number.

Reply:
Done.